keel-sql/src/main/resources/db/changelog/20201221-shorten-environment-last-verified-pk.yml

```diff
@@ -0,0 +1,30 @@
+databaseChangeLog:
+- changeSet:
+    id: shorten-environment-last-verified-pk
```

Comment:
It doesn't make a lot of sense to include `artifact_version` in the PK for this table as it's always just the `CURRENT` version in the environment.